suse_migration_services/units/mount_system.py

```diff
@@ -118,17 +119,10 @@ def initialize_logging():
         log.set_logfile(Defaults.get_migration_log_file())
 
 
-def initialize_debugging():
-    debug_file = Defaults.get_system_migration_debug_file()
-    # delete potentially existing debug flag file from migration live system
-    Path.wipe(
-        os.sep.join(['/etc', os.path.basename(debug_file)])
-    )
-    # copy debug flag file if set on target system to migration live system
-    if os.path.exists(debug_file):
-        Command.run(
-            ['cp', debug_file, '/etc']
-        )
+def update_config_info():
+    # update the default configuration info with provided data, if any
+    if os.path.exists(Defaults.get_system_migration_opt_config_file()):
+        MigrationConfig().update_config_info()
```

Comment:
I think this method belongs to the MigrationConfig class.

Reply:
Fixed the name and now it calls directly to MigrationConfig method